gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/AuditResource.java

```diff
@@ -69,28 +69,23 @@ public class AuditResource extends AbstractResource {
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     @Permissions(
-            {
-                    @Permission(value = RolePermission.ENVIRONMENT_AUDIT, acls = RolePermissionAction.READ),
-                    @Permission(value = RolePermission.ORGANIZATION_AUDIT, acls = RolePermissionAction.READ),
-            }
+        {
+            @Permission(value = RolePermission.ENVIRONMENT_AUDIT, acls = RolePermissionAction.READ),
+            @Permission(value = RolePermission.ORGANIZATION_AUDIT, acls = RolePermissionAction.READ),
+        }
     )
     public AuditEntityMetadataPage getAudits(@BeanParam AuditParam param) {
         AuditQuery query = new AuditQuery();
         query.setFrom(param.getFrom());
         query.setTo(param.getTo());
         query.setPage(param.getPage());
         query.setSize(param.getSize());
-        if (param.isEnvironmentLogsOnly()) {
```

Comment:
I would say yes but no. 
I have already thought about this subject and I came to this conclusion: 
As on the url 
`/management/organizations/DEFAULT/environments/DEFAULT/audit`
The fix of the bug is to add the scope by environment so having a filter isEnvironmentLogsOnly is inconsistent or it would have to add the bug again. 
On the other hand there is now a new feature for audit by organization. with a filter by environment. not available before. 
i think 🤔

Reply:
but can we put it in the release not ? or something like that. available to discuss